src/index.ts

```diff
@@ -327,6 +342,83 @@ export class FlashbotsBundleProvider extends providers.JsonRpcProvider {
     return this.sendRawBundle(signedTransactions, targetBlockNumber, opts)
   }
 
+  public async sendPrivateTransaction(
+    transaction: FlashbotsBundleTransaction,
+    opts?: {
+      maxBlockNumber?: number,
+      minTimestamp?: number,
```

Comment:
`minTimestamp` is not a supported option for sendPrivateTx, only `maxBlockNumber` is

Reply:
right, it's only used for the simulation callback, not sent to the relay. 
```js
const params = {
  tx: signedTransaction,
  maxBlockNumber: opts?.maxBlockNumber,
}
const request = JSON.stringify(this.prepareRelayRequest('eth_sendPrivateTransaction', [params]))
```

Reply:
actually, @metachris what do you think of putting the `minTimestamp` param in the `simulate` args instead?

```js
const minTimestamp: number | undefined = ...;
(await fbProvider.sendPrivateTransaction(...)).simulate(minTimestamp)
```

I like this better

Reply:
well... only thing I don't like about it is that it's inconsistent with the `sendBundle` interface. I'd prefer the `sendBundle` and `sendPrivateTransaction` callbacks to match one another

Reply:
I'm not sure, kind of feel like `sendPrivateTransaction` should only have the arguments that are actually used for `sendPrivateTransaction`. At the very least, comments for the opts might help users know what's going on.

Reply:
100% agree on comments. I'll add some.

On the minTimestamp issue, what if we just change `minTimestamp` to `simulationTimestamp`? That way it's obvious that it's not for the relay call, but it also doesn't change the `result.simulate()` signature.

Reply:
sounds good 👍

Reply:
@metachris done ✅